src/scene/light.js

```diff
@@ -393,7 +405,8 @@ class Light {
         // 16 - 17 : cookie channel G
         // 14 - 15 : cookie channel B
         // 12      : cookie transform
-        //  9 - 11 : light source shape
+        // 10 - 11 : light source shape
+        //  7 -  9 : light num cascades
```

Comment:
Couldn't this fit in 2 bits if you minus 1 from `numCascades`? Or is that not worth saving?

Reply:
it can never hurt to save a bit. done, thanks!